content/python/concepts/functions/terms/isinstance/isinstance.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'isinstance()'
+Description: 'Returns `True` if object is the specified type. Otherwise function will return `False`.'
+Subjects:
```

Comment:
This needs to be edited. Subjects should only come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).